tap/cleaner.go

```diff
@@ -72,25 +66,3 @@ func (cl *Cleaner) dumpStats() CleanerStats {
 
 	return stats
 }
-
-func deleteOlderThan(matcherMap *sync.Map, t time.Time) int {
```

Comment:
how can we remove this logic of deletion? something else is handling the cleaning?

Reply:
@gadotroee with this change the matcher maps created and destroyed for each TCP stream. So for short-living TCP connections the matcher maps are short-living. But for long-living TCP connections, if they are very long, there is no drop mechanism for unmatched requests or responses implemented in this PR. I'm not sure if there is a need for that.

With this PR there are 3 matcher maps for each TCP stream. So the number of iteration scales with the number of TCP streams. But the `clean` function is triggered every 10 seconds. Therefore I don't know.

Reply:
@gadotroee with https://github.com/up9inc/mizu/commit/6363d6392a5f369f384238cadf57332023ebf2a4 I brought back the  `deleteOlderThan` method.